includes/adapter/RecipientFromWCOrder.php

```diff
@@ -67,10 +67,24 @@ private function getAddressFromOrder(WC_Order $order, string $type): array
         $street       = WCX_Order::get_meta($order, "_{$type}_street_name") ?: null;
         $number       = WCX_Order::get_meta($order, "_{$type}_house_number") ?: null;
         $numberSuffix = WCX_Order::get_meta($order, "_{$type}_house_number_suffix") ?: null;
-        $addressLine1 = $this->separateStreet($order->{"get_{$type}_address_1"}(), $order, $type);
         $addressLine2 = $order->{"get_{$type}_address_2"}();
+        $isNL         = $order->{"get_{$type}_country"}() === AbstractConsignment::CC_NL;
+        $isBE         = $order->{"get_{$type}_country"}() === AbstractConsignment::CC_BE;
 
         $isUsingSplitAddressFields  = ! empty($street) || ! empty($number) || ! empty($numberSuffix);
+
+        if (! $isNL && ! $isBE) {
+            $fullStreet = $isUsingSplitAddressFields ? implode(' ', [$street, $number, $numberSuffix])
+                : $order->{"get_{$type}_address_1"}();
+
+            return [
+                'full_street'            => $fullStreet,
+                'street_additional_info' => $addressLine2 ?? null,
+            ];
+        }
+
+        $addressLine1 = $this->separateStreet($order->{"get_{$type}_address_1"}(), $order, $type);
```

Comment:
Het is m.i. begrijpelijker om `$addressLine1 = $order->{"get_{$type}_address_1"}();` te zetten voor $addressline2 zodat je die ook kunt gebruiken in de `$fullStreet = $iUsingSplitAdd[...]` en de $addressLine1 hier `$streetParts` te noemen ofzo, het is tenslotte en array (? blijkbaar). Deze vind ik wel reden tot afkeur want dat $addressLine1 een array is vind ik niet evident. Sorry dat ik dat niet eerder gezien heb.